### ISSUE TITLE
Already disposed error on showing files

### ISSUE THREAD

**Issue by jontwo** [hsz/idea-gitignore#539]
### Prerequisites

* [x] Plugin is in the latest version
* [x] Issue was not reported yet
* [x] Stack trace (if provided) contains `mobi.hsz.idea.gitignore` package name

### Description

I got the popup that said "Ignored files found that are tracked" and clicked Show details, but it threw an error.

Stack trace:

```
Already disposed: Project (Disposed) gdal
java.lang.AssertionError: Already disposed: Project (Disposed) gdal
	at com.intellij.openapi.components.impl.ComponentManagerImpl.a(ComponentManagerImpl.java:247)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:43)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:927)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:47)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:43)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.b(ComponentManagerImpl.java:245)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.getPicoContainer(ComponentManagerImpl.java:238)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.getPicoContainer(ComponentManagerImpl.java:44)
	at com.intellij.project.ProjectKt.getStateStore(project.kt:17)
	at com.intellij.openapi.project.impl.ProjectImpl.getStateStore(ProjectImpl.java:177)
	at com.intellij.openapi.project.impl.ProjectImpl.getBaseDir(ProjectImpl.java:209)
	at mobi.hsz.idea.gitignore.ui.untrackFiles.UntrackFilesDialog.<init>(UntrackFilesDialog.java:133)
	at mobi.hsz.idea.gitignore.TrackedIgnoredFilesComponent$1.hyperlinkUpdate(TrackedIgnoredFilesComponent.java:126)
	at com.intellij.notification.impl.ui.NotificationsUtil$1.hyperlinkUpdate(NotificationsUtil.java:155)
	at javax.swing.JEditorPane.fireHyperlinkUpdate(JEditorPane.java:342)
	at javax.swing.text.html.HTMLEditorKit$LinkController.activateLink(HTMLEditorKit.java:875)
	at javax.swing.text.html.HTMLEditorKit$LinkController.mouseClicked(HTMLEditorKit.java:674)
	at java.awt.AWTEventMulticaster.mouseClicked(AWTEventMulticaster.java:270)
	at java.awt.AWTEventMulticaster.mouseClicked(AWTEventMulticaster.java:269)
	at java.awt.AWTEventMulticaster.mouseClicked(AWTEventMulticaster.java:269)
	at java.awt.Component.processMouseEvent(Component.java:6544)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3325)
	at java.awt.Component.processEvent(Component.java:6306)
	at java.awt.Container.processEvent(Container.java:2237)
	at java.awt.Component.dispatchEventImpl(Component.java:4897)
	at java.awt.Container.dispatchEventImpl(Container.java:2295)
	at java.awt.Component.dispatchEvent(Component.java:4719)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4889)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4535)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4467)
	at java.awt.Container.dispatchEventImpl(Container.java:2281)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4719)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.awt.EventQueue$4.run(EventQueue.java:735)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.q(IdeEventQueue.java:779)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:716)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:395)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

### Versions

**Plugin:** 

```
2.6.1
```

**IDE:** 

```
PyCharm 2018.1.2 (Professional Edition)
Build #PY-181.4668.75, built on April 25, 2018
```

**OS:** 

```
Windows 10 10.0
```

### Additional Information

```
JRE: 1.8.0_152-release-1136-b29 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

**Comment by hsz** [comment]
@jontwo  Thanks for this report. It's fixed and released with v2.6.2

**Comment by jontwo** [comment]
Great, thanks!
